kitsune/wiki/migrations/0013_alter_document_related_documents_and_more.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 4.1.3 on 2022-11-22 12:09
+
+from django.conf import settings
```

Comment:
What's causing this migration?

Reply:
I believe this was the django-taggit update to 3.0.0.